### PR TITLE
build(aio): PWA improvements/fixes

### PR DESCRIPTION
**Pull request by gkalpak**
- Upgrade `lighthouse` to v1.6.5.
- ~~Remove unnecessary `pretest-pwa-score-local` script.~~
- Fix PWA testing on Windows (and reduce flaky-ness on CI). <-- for @wardbell with ❤️ 😛

### CODE REVIEW

File: aio/scripts/test-pwa-score.js [hunk 1]
```diff
@@ -64,7 +64,6 @@ function getScore(results) {
 
 function ignoreHttpsAudits(aggregations) {
   const httpsAudits = [
-    'is-on-https',
```
Comment (gkalpak) [inline aio/scripts/test-pwa-score.js]: Lighthouse v1.6.5 treats `localhost`/`1.2.7.0.0.1` as secure domains (i.e. as if they where HTTPS), so we need to stop handling the `is-on-https` audit specially.

**Review by petebacondarwin — APPROVED** [review]
I guess we need @IgorMinar to cherry pick this one too, rather than merging?

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
Hold on... something's not right:

```
Lighthouse PWA score:
[1]   - Expected: 90 / 100 (or higher)
[1]   - Actual:   19 / 100
```

**Comment by gkalpak** [comment]
It seems that `yarn e2e` deletes the `dist/` directory!?!

**Comment by gkalpak** [comment]
I had to revert one commit (because of https://github.com/angular/angular/pull/16814#issuecomment-301738691), but now it is green. From a quick look =, this might not affect @IgorMinar's PR, but I'm not sure.

File: aio/scripts/test-pwa-score.js [hunk 2]
```diff
@@ -83,6 +82,7 @@ function launchChromeAndRunLighthouse(url, flags, config) {
 
   return launcher.run().
     then(() => lighthouse(url, flags, config)).
+    then(results => new Promise(resolve => setTimeout(() => resolve(results), 1000))).
```
Comment (IgorMinar) [inline aio/scripts/test-pwa-score.js]: why the delay? can you at least add a comment?

File: aio/package.json [hunk 3]
```diff
@@ -86,7 +86,7 @@
     "karma-coverage-istanbul-reporter": "^0.2.0",
     "karma-jasmine": "~1.1.0",
     "karma-jasmine-html-reporter": "^0.2.2",
-    "lighthouse": "^1.6.3",
+    "lighthouse": "^1.6.5",
```
Comment (IgorMinar) [inline aio/package.json]: this will create a merge conflict for me, so I'll cherry-pick this into my branch

**Review by IgorMinar — CHANGES_REQUESTED** [review]

File: aio/scripts/test-pwa-score.js (hunk 2 above)
Comment (gkalpak) [inline aio/scripts/test-pwa-score.js]: I had the explanation in the [commit message][1]. I added a comment too.

[1]: https://github.com/angular/angular/pull/16814/commits/04c9e711e2eab20e23d1bf1bd1b33f3dd071f0af

File: aio/package.json (hunk 3 above)
Comment (gkalpak) [inline aio/package.json]: For posterity:
As discussed "offline", this PR just improve reliability (it doesn't improve the score or anything).
So, feel free to either cherry-pick this into your PR or refrain from updating lighthouse in your PR (both options should work equally well for your pusposes).

**Comment by gkalpak** [comment]
Note: This should not be merged before #16665. (It will be either cherry-picked into #16665 or merged after that.)

**Comment by petebacondarwin** [comment]
Cherry picked into #16665

**Pull request closed without merging**